# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderBridgeParent.h
# Commit: 5506d1803e15
# Full Hash: 5506d1803e158df5504bc74a7818cb78dca6420c
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-10-16 05:20:00
# Description:
#   Bug 1585801 - Fixing crash when folder is not writable and refactored logic. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D48999
# ==============================================================================

diff -r f24ddb55b48b -r 5506d1803e15 gfx/layers/wr/WebRenderBridgeParent.h
--- a/gfx/layers/wr/WebRenderBridgeParent.h	Sat Oct 12 23:50:45 2019 +0000
+++ b/gfx/layers/wr/WebRenderBridgeParent.h	Tue Oct 15 17:03:22 2019 +0000
@@ -151,6 +151,7 @@
   mozilla::ipc::IPCResult RecvClearCachedResources() override;
   mozilla::ipc::IPCResult RecvScheduleComposite() override;
   mozilla::ipc::IPCResult RecvCapture() override;
+  mozilla::ipc::IPCResult RecvSetTransactionLogging(const bool&) override;
   mozilla::ipc::IPCResult RecvSyncWithCompositor() override;
 
   mozilla::ipc::IPCResult RecvSetConfirmedTargetAPZC(